sql/mysqld.cc

```diff
@@ -8820,6 +8820,28 @@ static int get_options(int *argc_ptr, char ***argv_ptr)
   return 0;
 }
 
+/*
+  This function uses the maturity codes from _ and returns the respective maturity string as in
```

Comment:
We are trying to stick to 80 char rule, this line should have a break after ` respective`.